be/src/io/cache/block/block_file_cache_settings.h

```diff
@@ -24,6 +24,8 @@
 
 #include "io/cache/block/block_file_cache_fwd.h"
 
+#include <array>
```

Comment:
Where is the include used

Reply:
https://github.com/apache/doris/pull/19199/files like this